provider/file_template.rb

```diff
@@ -100,11 +100,16 @@ class ProviderFileTemplate < Provider::FileTemplate
     # All the products that are being compiled with the provider on this run
     attr_accessor :products
 
-    def initialize(output_folder, version, env, products)
+    # Optional path to the directory where overrides reside. Used to locate files
+    # outside of the MM root directory
+    attr_accessor :override_path
```

Comment:
It doesn't look like this ever gets used, does it?

Reply:
Not within MM currently. When specifying an override path that contains a `common~compile.yaml` a user can access `override_path` within that file in ERB, letting them compile files that are not in the MM directory